backend/src/data/repositories/master/master.repository.ts

```diff
@@ -40,17 +40,19 @@ class Master {
       return null;
     }
 
-    return Master.modelToEntity(master);
+    return MasterRepository.modelToEntity(master);
   }
 
   async create({
     master,
     passwordHash,
     passwordSalt,
+    tenantId,
```

Comment:
I know that it is not a part of your task but please remove all this properties and pass just MasterEntity object to create:
`async create(master: MasterRntity)`. MasterEntity should already contains all required fields.
Main idea behind Entities is that repositories should **depend** on Entities and Services. More information about it you can read [here](https://t2pdev.com/dependency-inversion-and-the-clean-architecture/). In current implementation service depends on reposiotry and it is violates Dependency Inversion principle. Also it violates DRY because we duplicate properties in both Entity and create method